airmozilla/main/templates/main/_events.html

```diff
@@ -19,7 +19,10 @@ <h2 class="entry-title">
       {% endif %}
     </a>
 
-    <a class="star{% if star_on %} star-on{% endif %}" data-id="{{ event.id }}" title="Save as a starred event"></a>
+    <a class="star{% if star_on %} star-on{% endif %}" data-id="{{ event.id }}" 
```

Comment:
This will also need to use the helper function but I wasn't sure the best way to deal with the if statement.